src/clique_config.erl

```diff
@@ -233,97 +268,43 @@ run_callback({Args, Flags, Status}) ->
                                      Acc
                              end
                          end, [], Args),
-    _ = [F(K, V, Flags) || {K, V, F} <- KVFuns],
-    Status.
-
--spec get_config(err()) -> err();
-                ({args(), flags()}) ->
-                    {proplist(), proplist(), flags()} | err().
-get_config({error, _}=E) ->
-    E;
-get_config({[], _Flags}) ->
+    OutStrings = [F(K, V) || {K, V, F} <- KVFuns],
```

Comment:
Could maybe be simplified by interleaving ets lookups with callbacks in one list comprehension?

OutStrings = [F(K, V) || {K, V} <- Args, {K, F} <- ets:lookup(?config_table, K)]